parcimonie.sh

```diff
@@ -140,12 +140,16 @@ getNumKeys() {
 }
 
 getRandomKey() {
-	local allPublicKeys fingerprint
+	local allPublicKeys fingerprint randomChoice
 	allPublicKeys=()
 	for fingerprint in $(getPublicKeys); do
 		allPublicKeys+=("$fingerprint")
 	done
-	echo "${allPublicKeys[$(expr "$(getRandom)" % "${#allPublicKeys[@]}")]}"
+	randomChoice=$(expr "$(getRandom)" % "${#allPublicKeys[@]}")
+	if [ "$randomChoice" -lt 0 ]; then
+		randomChoice=$(expr "$randomChoice" "*" "-1")
```

Comment:
As per https://github.com/EtiennePerot/parcimonie.sh/pull/18#discussion_r43345109, needs `'` instead of `"` in strings that don't have variables in them. Same below.